relay-profiling/src/native_debug_image.rs

```diff
@@ -49,6 +49,7 @@ mod tests {
                 code_id: Annotated::empty(),
                 debug_file: Annotated::empty(),
                 debug_id: Annotated::new("32420279-25E2-34E6-8BC7-8A006A8F2425".parse().unwrap()),
+            debug_checksum: Annotated::empty(),
```

Comment:
something wrong with formatting here?

Reply:
Interesting. CI is not complaining, and my local rustfmt refuses to automatically fix this, so I did manually.